admin/app/components/users/certification-centers/membership-item.js

```diff
@@ -0,0 +1,42 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+
+export default class UsersCertificationCentersMembershipItemComponent extends Component {
+  @service notifications;
+  @service store;
+  @service intl;
+
+  @tracked isEditionMode = false;
+
+  get certificationCenterRoles() {
+    return [
+      { value: 'ADMIN', label: this.intl.t('common.roles.admin') },
+      { value: 'MEMBER', label: this.intl.t('common.roles.member') },
+    ];
+  }
```

Comment:
La valeur de `certificationCenterRoles` ne change jamais et donc il ne sert à rien de la réévaluer à chaque utilisation avec une fonction, l'intérêt d'un _getter_ étant d'être réévalué à chaque appel.

Je suggère donc de gérer cette valeur sous forme d'une _variable_ au lieu d'un _getter_. Et comme il s'agit d'options d'un select, je suggère de mettre le mot « options » dans le nom de cette variable et de ne pas repréciser le mot « certificationCenter », ce qui donne la suggestion ci-dessous :

```suggestion
  rolesOptions = [
      { value: 'ADMIN', label: this.intl.t('common.roles.admin') },
      { value: 'MEMBER', label: this.intl.t('common.roles.member') },
    ];
```